docs/src/main/asciidoc/ide-tooling.adoc

```diff
@@ -0,0 +1,157 @@
+
+:vscode-logo: https://simpleicons.org/icons/visualstudiocode.svg 
+:eclipse-logo: https://simpleicons.org/icons/eclipseide.svg
+:intellij-logo: https://simpleicons.org/icons/intellijidea.svg
+:che-logo: che-icon-dark.svg
+[cols="6*^", header]
+|===
+| .
+| image:{vscode-logo}[200,200]
+{empty} +
+VSCode Quarkus Tools
+| image:{eclipse-logo}[200,200]
+{empty} +
+Eclipse Quarkus Tools
+| image:{intellij-logo}[100,100]
+{empty} +
+IntelliJ Quarkus Tools
+| image:{intellij-logo}[100,100]
+{empty} +
+Intellij Ultimate
+| image:{che-logo}[100,100]
+{empty} +
+Eclipse Che
+
+|Description
+|Visual Studio Code extension to install using the marketplace  
+|Eclipse plugin to install into Eclipse using an updatesite
+|Intellij plugin that works in Intellij Community and Ultimate. Available from Marketplace.
+|Built-in Quarkus features available only in IntelliJ Ultimate
+|Built-in Quarkus features available in Eclipse Che incl. che.openshift.io.
+
+|Status
+|Stable
+|Alpha
+|Beta
+|Stable (since 2019.3)
+|Coming Soon!
+
+|Downloads
+| https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[Marketplace]
+{empty} +
+ https://download.jboss.org/jbosstools/vscode/snapshots/vscode-quarkus/?C=M;O=D[Development Builds]
+| https://download.jboss.org/jbosstools/photon/snapshots/builds/jbosstools-quarkus_master/[Development Update Site]
+| https://plugins.jetbrains.com/plugin/13234-quarkus/versions[Marketplace]
+{empty} +
+https://download.jboss.org/jbosstools/intellij/snapshots/intellij-quarkus/[Development Builds]
+| https://www.jetbrains.com/idea/nextversion/[Installer]
+|
+
+|Source
+|https://github.com/redhat-developer/vscode-quarkus[Github]
+|https://github.com/jbosstools/jbosstools-quarkus[Github]
+|https://github.com/redhat-developer/intellij-quarkus[Github]
+|Closed-Source
+|
+
+|https://github.com/redhat-developer/quarkus-ls[Quarkus Language Server]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:times[]
+|icon:check[]
+
+|Wizards w/code.quarkus.io
+|icon:check[]
+|icon:check[]
+|https://issues.jboss.org/browse/JBIDE-26950[icon:times[]]
+|icon:check[]
+|icon:check[]
+
+|Custom Wizard
+|icon:times[]
+|icon:times[]
+|icon:check[]
+|icon:check[]
+|icon:times[]
+
+|Config editor
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:times[]
+|icon:check[]
+
+|Config autocompletion
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+
+|Config validation
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+
+|Config profiles
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+
+|Config outline 
+|icon:check[]
+|icon:check[]
+|icon:check[]
+|icon:times[]
+|icon:check[]
+
+|Easy Launch debug/dev:mode
+|icon:check[]
+|icon:times[]
+|icon:check[]
+|icon:check[]
+|icon:check[]
+
+|Quarkus Code Snippets
+|icon:check[]
+|icon:times[]
+|icon:times[]
+|icon:times[]
+|icon:check[]
+
+|Injection Discovery/Navigation
```

Comment:
What do you mean about this feature? Could you explain more what is this feature please?

Reply:
intellij has features similar to what we had/have in jboss tools around find references like feature to navigate from a @Injection and similar annotations to possible sources and other way around.

Reply:
Thanks @maxandersen for your clarification. Do you think LSP4MP should support this feature?

Reply:
if feasible - there are quite the special cases to cover and getting UX for it working would be important. in jboss tools very few realized it was even there so almost not used.